[UPDATED] Updated TOS Structure

AadilRayeen opened the pull request:
Fixing the updated issue https://github.com/tcet-opensource/documentation/issues/202, the following things were accomplished:

1.Updated TCET Opensource Details and Description.
2.Updated the Organization according to new Structure enforced in July 2023.
3.Added Social Handles to Current Bearer in Docs.

Screenshots to support above fixes:

![Screenshot 2023-09-11 091004](https://github.com/tcet-opensource/documentation/assets/127043014/bdaa9efc-7653-4c55-848f-b7e3eb7ed8de)

vish-rt commented on docs/about-tcetopensource.mdx:
Technical documentation usually encompasses various concepts of a project with layman writing to reach out to your audience. When writing one, it should be plain, simple and follow some grammar.


```suggestion
**TCET Open Source** is an organization where software is not just a commodity to be bought and sold, but a collaborative effort that brings people together to solve problems and build something greater than themselves. Our Organization crafts solutions with the spirit of open source in it's heart, encompassing collaborative development and community involvement while building projects.
TCET Open Source is more than just a movement, it's a philosophy that values transparency, inclusivity, and the power of community, having a strength of **150+ Individuals** currently, across various teams.
As stewards of the `digital landscape`, We embark on a **journey** where every contribution shapes a **brighter future**. 
```

Above example describes basic grammar usage with minimal styling, which would be easy to read.

vish-rt reviewed (approved):
I suggest taking notes from the comment and try to keep the page easy and less distractive to read.

vish-rt reviewed (changes requested):
Refer to any other comments by fellow maintainers before merging.

AadilRayeen commented:
Fixing the updated issue https://github.com/tcet-opensource/documentation/issues/202, the following things were accomplished:

1. Updated TCET Opensource Details and Description as per requested changes.
2. Updated the Organization according to the new Structure enforced in September 2023.


Screenshots to support the above fixes:

New Description :-
![Screenshot 2023-09-20 211156](https://github.com/tcet-opensource/documentation/assets/127043014/f2a23551-99d5-4bcc-97d2-bc09b23d5dd5)


Updated Image of Org Stucture :-
![Screenshot 2023-09-20 211234](https://github.com/tcet-opensource/documentation/assets/127043014/6590405e-777a-4735-a474-5c5f4fd9ba6b)

vish-rt commented:
In Docusaurus, if you try to highlight any placeholders (for when you are unsure what should be in the place), it should not be left within the codebase as the site would not build with broken links.

Leaving an example for you:
```md
**Current Bearer - [a](a)**
```
For any markdown-based website, utilizing hyperlink parenthesis `[ ]( )` without a valid webpage link indicates self routing within the website. Hence, the above code should lead to **/docs/a**, which does not exist and leads to build fail.

Serving a hotfix for it in an upcoming commit.

AadilRayeen commented:
Fixing the updated issue https://github.com/tcet-opensource/documentation/issues/202, the following things were accomplished:

1. Updated Operations Teams Role Bearers

Screenshots to support the above fixes:

![Screenshot 2023-09-26 213356](https://github.com/tcet-opensource/documentation/assets/127043014/1f969682-09ec-4118-9da4-317c763e73ff)

aryadubey03 reviewed (changes requested):
In the COO's description, second sentence feels too long. It can be written in a more readable way. Do these changes where similar long sentences are used. Also, check punctuation properly.

himanshu-03 commented:
@AadilRayeen any updates?

AadilRayeen commented:
Fixing the updated issue https://github.com/tcet-opensource/documentation/issues/202, the following things were accomplished:

Updated the data as per requested.

NikitaSShukla reviewed (approved):
The content is well written! There are some minor grammatical mistakes, please correct it as soon as possible.

AadilRayeen commented:
Fixed grammatical errors !!!
